docs/docs/user-section/customization-and-configuration.md

```diff
@@ -10,3 +10,14 @@ custom_edit_url: null
 
 ---
 
```

Comment:
@Alsardy We still don't have this functionality but when we have it, its better to be specific on how the user can do that, no just that Sakundi allows that somehow.